src/main/java/org/example/service/HelloService.java

```diff
@@ -0,0 +1,18 @@
+package org.example.service;
+
+
+public class HelloService {
+
+    private static HelloService INSTANCE = null;
```

Comment:
Remove null.

Check the Java conventions for naming:

- variables
- methods
- classes
- constant variables
- class files

Reply:
fix